esx_service/utils/auth_data_test.py

```diff
@@ -37,253 +37,259 @@ def setUp(self):
         self.auth_mgr.connect()
         self.auth_mgr.create_tables()
 
-   #def tearDown(self):
-       #os.unlink(self.db_path)
+   def tearDown(self):
+       os.unlink(self.db_path)
 
-    # def test_create_tenant(self):
-    #     vms = [(str(uuid.uuid4()), 'vm1')]
-    #     privileges = [{'datastore': 'datastore1',
-    #                  'global_visibility': 0,
-    #                   'create_volume': 0,
-    #                   'delete_volume': 0,
-    #                  'mount_volume': 0,
-    #                   'max_volume_size': 0,
-    #                   'usage_quota': 0}]
-    #     #privileges = []
-    #     default_datastore = 'default_ds'
-    #     default_privileges = [{'datastore': default_datastore,
-    #                           'global_visibility': 0,
-    #                           'create_volume': 0,
-    #                           'delete_volume': 0,
-    #                           'mount_volume': 0,
-    #                           'max_volume_size': 0,
-    #                           'usage_quota': 0}]
-    #     tenant1 = self.auth_mgr.create_tenant('tenant1', 'Some tenant', default_datastore,
-    #                                           default_privileges, vms, privileges)
-    #     self.assertTrue(uuid.UUID(tenant1.id))
+    def test_create_tenant(self):
+        vms = [(str(uuid.uuid4()), 'vm1')]
```

Comment:
All this boilerplate to create default privileges/datastore and a tenant can be abstracted into it's own function and used in each test.